rules/windows/command_and_control_teamviewer_remote_file_copy.toml

```diff
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/09/02"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies an executable or script file remotely downloaded via a TeamViewer transfer session, verify the context of
+this activity.
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Remote File Copy via TeamViewer"
+references = ["https://blog.menasec.net/2019/11/hunting-for-suspicious-use-of.html"]
+risk_score = 47
+rule_id = "b25a7df2-120a-4db2-bd3f-3e4b86b24bee"
+severity = "medium"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:file and event.type:creation and
+ process.name:TeamViewer.exe and
+ file.name:(*.exe or *.dll or *.scr or *.com or *.bat or *.ps1 or *.vbs or *.vbe)
```

Comment:
should be able to use file.extension here instead

Reply:
converted